x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportDeleteFilterAction.java

```diff
@@ -44,8 +44,7 @@
     public TransportDeleteFilterAction(TransportService transportService,
                                        ActionFilters actionFilters, Client client,
                                        JobConfigProvider jobConfigProvider) {
-        super(DeleteFilterAction.NAME, transportService, DeleteFilterAction.Request::new, actionFilters
-        );
+        super(DeleteFilterAction.NAME, transportService, DeleteFilterAction.Request::new, actionFilters);
```

Comment:
Swap this super ctor too?

Reply:
nope, surprisingly this class was not converted to Writeable yet.